R/apa_print_nlme.R

```diff
@@ -0,0 +1,133 @@
+#' Format Statistics from Nonlinear Mixed-Effects Models (APA 6th edition)
+#'
+#' These methods take mixed-effects models fitted with the \pkg{nlme} package
+#' and create formatted character strings report the results in accordance
+#' with APA manuscript guidelines.
+#'
+#' @param x A (non-)linear mixed-effects model fitted with [nlme::lme()] or [nlme::nlme()]
+#' @param args_confint Named list. Arguments that are passed to [nlme::intervals.lme()].
+#' @inheritParams apa_print.merMod
+#' @examples
+#'   library(nlme)
+#'   fm1 <- lme(distance ~ age, data = Orthodont, method = "ML") # random is ~ age
+#'   fm2 <- lme(distance ~ age + Sex, data = Orthodont, random = ~ 1, method = "ML")
```

Comment:
You fit three models here, but only the first one ever gets used in the example. Maybe drop the two others?